tutu-frontend/src/app/hakemus/[oid]/perustelu/yleiset/perustelut/components/SelvitysTutkinnonMyontajastaJaVirallisuudesta.tsx

```diff
@@ -0,0 +1,20 @@
+import { useHakemus } from '@/src/context/HakemusContext';
+import { useTranslations } from '@/src/lib/localization/hooks/useTranslations';
+
+import { Muistio } from '@/src/components/Muistio';
+
+export const SelvitysTutkinnonMyontajastaJaVirallisuudesta = () => {
```

Comment:
ei ehkä tartte uutta komponenttiä jos näitä ei oo muualla käytössä page:lle vaan muistio